readme.md

```diff
@@ -104,6 +104,15 @@ Sets `options.method` to the method name and makes a request.
 
 Type: `Object`
 
+##### method
+
+Type: `String`
+Default: `'GET'`
+
+Sets the method name and makes the request.
```

Comment:
This is not correct. It doesn't make the request. And "sets the method name" is not really more descriptive than the option name.

It could say:

> Set the HTTP request method.

Reply:
Also I think it should be: `'GET'` => `GET`